src/components/ui/Button.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+      <button @click="handleClick">{{ label }}</button>
```

Comment:
Instead of label you can use slots here. It will make your component more general. See it here: https://vuejs.org/guide/components/slots.html

You dont need to emit a click event since its a fallthrough attribute.